### ISSUE TITLE
moving docker setup to alpine

### ISSUE THREAD

**Issue by sudhackar** [SwiftLaTeX/SwiftLaTeX#3]
Why use ubuntu which on installing does this?

```
Need to get 126 MB of archives.
After this operation, 440 MB of additional disk space will be used.
```

Instead use alpine which is small and just contains what's needed. I have tested this dockerfile

```
FROM python:3.8-alpine
RUN apk add --no-cache git gcc libc-dev linux-headers

RUN git clone https://github.com/SwiftLaTeX/SwiftLaTeX.git /app && \
    pip3 install -r /app/requirements.txt && echo "0.1"

WORKDIR /app
CMD ["python3", "wsgi.py"]
```

Change the repo url to https to make it work without ssh. 

If OK I can send a PR.

**Comment by elliott-wen** [comment]
Hi Sudhakar.

Yes. Thanks a lot.

Cheers
Elliott

**Comment by sudhackar** [comment]
closed via #4
